src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -10,12 +10,66 @@
 
 #define THIS ((EnIk*)thisx)
 
+typedef enum {
+    /* 0 */ IK_TYPE_SILVER,
+    /* 1 */ IK_TYPE_BLACK,
+    /* 2 */ IK_TYPE_WHITE
+} EnIkType;
```

Comment:
This should be in the header if it could be used externally.

Reply:
There's no real use of it outside of the actor, I think it makes sense to keep it in the c file

Reply:
I'd have thought the actor lists in the scene files might want it eventually.

Reply:
There was no usage from what I saw. What is used in the the scene files the `IK_GET_ARMOR_TYPE` params macro in the header takes care of.

Reply:
Except that takes `thisx`, which of course is not available outside an actor. It'll end up being some sort of packing macro, most likely. But you need to have something to describe the types there anyway, right?